11ty/_includes/components/alertbox.njk

```diff
@@ -0,0 +1,4 @@
+<div class="alertbox">
+  <h2 class="alertbox__title">Something not quite right?</h2>
+  <p class="alertbox__content">If you found a typo, would like to add another resource, or add nuance to this definition, please feel free to submit <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferral">an issue</a> or <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull request</a>.</p>
```

Comment:
Hey @sarenji , these are great suggestions! 

Good catch on `noreferral`. I copied the links from the ones already in `/selfdefined/11ty/index.njk` without thinking about the syntax. I don't have any background on why `noreferral` would have been used as the `rel` value for those links but operating under the assumption that there's a reason that file's author put it there and that they actually wanted `noreferrer`, I've updated the syntax there as well as in my file.

I've taken your suggestion about including the action verbs in the links as well; it reads and scans better. Thanks!

Reply:
The author was me and the reason was «not really thinking». Thanks for fixing it. If you like, feel free to add them to other external links, too. If not, no worries, I’ll open a separate PR :)

Reply:
Hey @olvb , thanks for the context! 

I don't mind making that update but I'm thinking it might be good to address updating other external links in a separate PR so that we can get this callout into prod after being in the works for a few weeks and keep this PR lean.

If I were to submit a separate PR for that, should all external links have `rel=noreferrer` applied to them?

Reply:
Agreed with not cluttering this PR. I’ve created #236 to have a dedicated space for this discussion and to agree on an outcome which we can implement.

Reply:
(Oh, btw, my username is ovlb, olvb is someone totally unrelated, but I think by now they have muted this repo, because this typo happens so often.)